synapse/storage/databases/main/stream.py

```diff
@@ -1475,7 +1475,7 @@ def _paginate_room_events_txn(
                 event.topological_ordering, event.stream_ordering
             FROM events AS event
             %(join_clause)s
-            WHERE event.outlier = ? AND event.room_id = ? AND %(bounds)s
+            WHERE event.outlier = FALSE AND event.room_id = ? AND %(bounds)s
```

Comment:
We need to remove `False` from `args = [False, room_id]` above here.

https://github.com/matrix-org/synapse/actions/runs/5471845973/jobs/9974994023?pr=15525#step:9:7190
```
  File "/home/runner/work/synapse/synapse/synapse/storage/databases/main/stream.py", line 1488, in _paginate_room_events_txn
    txn.execute(sql, args)
  File "/home/runner/work/synapse/synapse/synapse/storage/database.py", line 417, in execute
    self._do_execute(self.txn.execute, sql, parameters)
  File "/home/runner/work/synapse/synapse/synapse/storage/database.py", line 469, in _do_execute
    return func(sql, *args, **kwargs)
sqlite3.ProgrammingError: Incorrect number of bindings supplied. The current statement uses 3, and there are 4 supplied.
```